### DISCUSSION TITLE
Indented quote blocks

### DISCUSSION THREAD

**Discussion by mid-kid** [zim-desktop-wiki/zim-desktop-wiki#2619] [Ideas]
Sometimes I want to quote another piece of text, and I want this to appear in its own indented (and spaced out) block. I don't want it to appear like inline code nor be typeset in monospace. Is there any feature or plugin for something like this?

**Comment by sojusnik** [comment]
Not that I know of, but it would be awesome to have!

**Comment by jaap-karssenberg** [comment]
My advice would be to just use an indented block.

Regards,

Jaap

Op vr 31 mei 2024 15:58 schreef sojusnik ***@***.***>:
